.github/actions/publish_binary/databend-release-readme.txt

```diff
@@ -0,0 +1,15 @@
+Thank you for downloading Databend, an open-source elastic and workload-aware modern cloud data warehouse.
+
+You might need these links to move forward:
+- Deployment Instructions: https://databend.rs/doc/deploy
+- Documentation: https://databend.rs/doc
+- Code Browser: https://github.com/datafuselabs/databend
+
+We'd love to hear from you. If you have any questions about Databend or want to suggest a feature, reach out to us on GitHub: 
+- Issues: https://github.com/datafuselabs/databend/issues
+- Discussions: https://github.com/datafuselabs/databend/discussions
+
+Follow us on social media and stay tuned for more exciting updates on Databend:
+- Twitter：https://twitter.com/Datafuse_Labs
+- Slack：https://datafusecloud.slack.com/
```

Comment:
This link requires the user to have already joined slack account.

How about using https://link.databend.rs/join-slack instead?

Reply:
Done!